examples/ee-demo/app.js

```diff
@@ -71,7 +71,9 @@ export default class App extends React.Component {
     const layers = this.state.eeImage && [
       new EarthEngineLayer({
         eeObject: this.state.eeImage,
-        visParams: {min: 0, max: 255}
+        visParams: {min: 0, max: 255},
```

Comment:
One thing we need to consider is the diffing behavior of using nested props. deck.gl does shallow compare on layer props, so a new object with the same values will compare differently and if we are not careful could cause us to generate new urls.

Any change, if needed, can be done in separate PR.

Reply:
Yes, you're right. I'll work on this in a follow-up PR